include/common.hpp

```diff
@@ -35,7 +36,7 @@ using std::make_pair;
 using std::array;
 using std::vector;
 using std::to_string;
-
+using std::queue;
```

Comment:
Please remove this. We don't necessarily want to force everyone to include the `queue` header if they don't need it.

Reply:
Is it okay if I moved the include queue to controller.hpp? or just the using std::queue;